frontend/apps/reader/readerui.lua

```diff
@@ -369,6 +370,13 @@ function ReaderUI:init()
         view = self.view,
         ui = self,
     })
+    --  event listener for reader
+    self:registerModule("readerlistener", ReaderListener:new{
```

Comment:
Is it worth adding yet another module ? Or do you think you'll need that to add other stuff ?

Reply:
I would rather not have added one but togglereadingorder does not fit into any of the existing ones. any better ideas?

Reply:
Given that it's ReaderRolling and ReaderPaging that holds `self.inverse_reading_order` and handle page turning, I would just have that handled in each of them (so, duplicated, but it's consistent - and they are mutually exclusive).

Reply:
I agree that a little bit of duplication should be fine for this one, plus that's how it's been, hasn't it? (Albeit with some comment saying it shouldn't be.)

Reply:
fixed.
discussed earlier in #5423